client/src/site-search/advanced-search-options.tsx

```diff
@@ -11,66 +11,71 @@ const LANGUAGES = new Map(
   })
 );
 
-export default function SiteSearchForm() {
+export default function AdvancedSearchOptions() {
   const locale = useLocale();
   const [searchParams] = useSearchParams();
   const queryLocales = searchParams.getAll("locale");
+  const isBoth =
+    queryLocales.length === 2 && equalLocales(queryLocales, [locale, "en-us"]);
+  const isCurrentLocale =
+    !queryLocales.length ||
+    (queryLocales.length === 1 && equalLocales(queryLocales, [locale]));
+  const isEnglish =
+    queryLocales.length && equalLocales(queryLocales, ["en-us"]);
 
   return (
-    <div className="advanced-options">
-      {/* Language only applies if you're browsing in, say, French
-      and want to search in English too. */}
-      {locale.toLowerCase() !== "en-us" && (
-        <p className="advanced-option">
-          <b>Language</b>{" "}
-          {!queryLocales.length ||
-          (queryLocales.length === 1 &&
-            equalLocales(queryLocales, [locale])) ? (
-            <i>
-              {LANGUAGES.get(locale.toLowerCase())?.native} (
-              {LANGUAGES.get(locale.toLowerCase())?.English})
-            </i>
+    <>
+      <h2>Language</h2>
+      <ul className="search-language-options">
+        <li aria-current={isBoth ? true : false}>
+          {isBoth ? (
+            "Both"
```

Comment:
I don't like this. Please change it back to the way it was. 
<img width="451" alt="Screen Shot 2021-02-23 at 9 00 56 AM" src="https://user-images.githubusercontent.com/26739/108854269-adc79280-75b5-11eb-81c6-c813141dcb8c.png">

The default is that you search in the locale that is in your current URL. This is the default and it needs to be left-most. 
<img width="448" alt="Screen Shot 2021-02-23 at 9 01 45 AM" src="https://user-images.githubusercontent.com/26739/108854357-c3d55300-75b5-11eb-8b3a-b3b1edba6ef4.png">
When it's the middle choice, it looks like a choice has been made.